src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/locations/LocationsMigrationService.kt

```diff
@@ -232,9 +232,21 @@ fun toUpsertSyncRequest(nomisLocationResponse: LocationResponse, parentId: Strin
 
     createDate = nomisLocationResponse.createDatetime,
     // lastModifiedDate - no value available as it changes with occupancy
-    deactivatedDate = nomisLocationResponse.deactivateDate,
```

Comment:
do these have valves even when they the location is active?

Reply:
They get cleared in P-Nomis when you set the location active, but potentially that may not always happen. Will get trapped by the reconciliation.